schemas/external/schema/geoshape.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "http://schema.org/GeoShape",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Geo Shape",
+  "type": "object",
+  "description": "The geographic shape of a place. Based on [schema.org](http://schema.org/GeoShape).",
+  "definitions": {
+    "geoshape": {
+      "properties": {
+        "@id": {
+          "title": "Shape Id",
+          "type": "string",
+          "description": "The unique identifier of the shape."
+        },
+        "schema:description": {
+          "title": "Description",
+          "type": "string",
+          "description": "A description of what the shape is defining."
+        },
+        "schema:polygon": {
+          "title": "Polygon",
+          "type": "array",
+          "items": {
+            "$ref": "http://schema.org/GeoCoordinates"
+          },
+          "description": "A series of four or more coordinates where the first and final coordinates are identical. In schema.org, this is a plain text. In XDM, it is a structured array instead.",
+          "minItems": 4
+        },
+        "schema:circle": {
+          "title": "Circle",
+          "$ref": "http://schema.org/GeoCircle",
+          "description": "A circular region with a specific radius centered on a geographic coordinate."
+        },
+        "schema:box": {
+          "title": "Box",
+          "type": "array",
+          "items": {
+            "$ref": "http://schema.org/GeoCoordinates"
+          },
+          "description": "The area enclosed by the rectangle formed by two coordinates. The first coordinate is the lower corner and the second coordinate is the upper corner of a rectangle.",
+          "minItems": 2,
+          "maxItems": 2
+        },
+        "schema:elevation": {
+          "title": "Elevation",
+          "type": "number",
+          "description": "The specific or minimum altitude of the shape. This value conforms to the WGS84 datum and is measured in meters."
```

Comment:
`altitute` should be changed to `elevation` in the description.

Reply:
yep. Thanks for your vigilance.